backend/frontend/templates/frontend/page_course_component_details/base.html

```diff
@@ -0,0 +1,20 @@
+{% extends "frontend/base.html" %}
+
+{% load static %}
+
+{% block main_head %}
+    {{ block.super }}
+    <script src="{% static 'frontend/page_project_details.js' %}"></script>
+{% endblock %}
+
+{% block main_content %}
+  {% block content %}
+    <div class="mx-auto w-1/2 p-6 bg-white rounded my-4">
+      <h1 class="{{ styles.heading1 }}">
+        {% block project_heading %}{% endblock %}
```

Comment:
Is this base template only going be used for projects? Or topics as well? 

If the plan is to use it for both, then it would be best not to say "project_heading" here. Same for "project_content". Topics are not projects so the name would be misleading